web/migrations/0050_merge_20250329_1636.py

```
@@ -0,0 +1,14 @@
+# Generated by Django 5.1.6 on 2025-03-29 11:06
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('web', '0045_challengesubmission_is_public'),
+        ('web', '0049_session_latitude_session_longitude_and_more'),
+    ]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)